docs/set_up_admin_tails.rst

```diff
@@ -55,17 +55,14 @@ Start by running the following commands to download the git repository.
 Verify the Release Tag
 ~~~~~~~~~~~~~~~~~~~~~~
 
-First, download the *Freedom of the Press Foundation Master Signing Key*
-and verify the fingerprint.
+First, download and verify the *Freedom of the Press Foundation Master
+Signing Key*. When passing the full public key fingerprint to the
+:code:`--recv-key` command, GPG will implicitly verify that the
```

Comment:
Cool, I didn't know you could do this! I usually just use double backticks (aka backquotes) when I want an inline code span: http://sphinx-doc.org/rest.html#inline-markup.